### PR TITLE
Reorganized the logging sample

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

File: logging/src/ListEntriesCommand.php [hunk 1]
```diff
@@ -41,22 +38,15 @@ protected function configure()
...
+        $projectId = $input->getOption('project');
         $loggerName = $input->getOption('logger');
-        // [START list_log]
-        $logging = new LoggingClient(['projectId' => $project]);
-        $loggerFullName = sprintf('projects/%s/logs/%s', $project, $loggerName);
-        $options = [
-            'filter' => sprintf('logName = "%s"', $loggerFullName)
-        ];
-        foreach ($logging->entries($options) as $entry) {
+        foreach (list_entries($projectId, $loggerName) as $entry) {
```
Comment (bshaffer) [inline logging/src/ListEntriesCommand.php]: nit: put this on two lines for readability

```php
$entries =  list_entries($projectId, $loggerName);
foreach ($entries as $entry) {
    //...
}
```

File: logging/src/ListSinksCommand.php [hunk 2]
```diff
@@ -43,15 +40,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $projectId = $input->getOption('project');
         //$loggerName = $input->getOption('logger');
-        // [START list_sinks]
-        $logging = new LoggingClient(['projectId' => $projectId]);
-        foreach ($logging->sinks() as $sink) {
+        foreach (list_sinks($projectId) as $sink) {
```
Comment (bshaffer) [inline logging/src/ListSinksCommand.php]: nit: put this on two lines for readability

```php
$sinks =  list_sinks($projectId);
foreach ($sinks as $sink) {
```

File: logging/src/ListSinksCommand.php [hunk 3]
```diff
@@ -43,15 +40,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $projectId = $input->getOption('project');
         //$loggerName = $input->getOption('logger');
```
Comment (bshaffer) [inline logging/src/ListSinksCommand.php]: remove the comment!

File: logging/src/ListEntriesCommand.php (hunk 1 above)
Comment (tmatsuo) [inline logging/src/ListEntriesCommand.php]: Done

File: logging/src/ListSinksCommand.php (hunk 2 above)
Comment (tmatsuo) [inline logging/src/ListSinksCommand.php]: Done

File: logging/src/ListSinksCommand.php (hunk 3 above)
Comment (tmatsuo) [inline logging/src/ListSinksCommand.php]: Done

**Pull request merged**